README.md

```diff
@@ -25,6 +25,15 @@ pip install --upgrade jupyterlab-git
 jupyter lab build
 ```
 
+## Settings
+
+Once installed, extension behavior can be modified via the following settings which can be set in JupyterLab's advanced settings editor:
+
+-   **disableBranchWithChanges**: disable all branch operations, such as creating a new branch or switching to a different branch, when there are changed/staged files. Setting this setting to `true` guards against overwriting and/or losing uncommitted changes.
+-   **historyCount**: number of commits shown in the history log, beginning with the most recent. Displaying a larger number of commits can lead to performance degradation, so use caution when modifying this setting.
+-   **refreshInterval**: number of milliseconds between polling the file system for changes. In order to ensure that the UI correctly displays the current repository status, the extension must poll the file system for changes. Longer polling times increase the likelihood that the UI does not reflect the current status; however, longer polling times also incur less performance overhead.
+-   **simpleStaging**: enable a simplified concept of staging. When this setting is `true`, all files with changes are automatically staged. When we develop in JupyterLab, we often only care about what files have changed (in the broadest sense) and don't need to distinguish between "tracked" and "untracked" files. Accordingly, this setting allows us to simplify the visual presentation of changes, which is especially useful for those less acquainted with Git.
```

Comment:
Maybe we should include another gif or second binder here? I'll also note the top gif appears to be out of date.

Reply:
For reference, see https://github.com/jupyterlab/jupyterlab-git/pull/482. This stalled due to a bug and needs to be updated again to showcase more recent additions.

Reply:
Could either add a separate binder or update the current binder to provide a tutorial of sorts which guides the user into toggling the various settings and seeing their effects.